kogito-editors-js/packages/boxed-expression-component/src/components/Table/TableHandler.tsx

```diff
@@ -191,9 +191,22 @@ export const TableHandler: React.FunctionComponent<TableHandlerProps> = ({
         if (selectedColumn.appendColumnsOnChildren && _.isArray(selectedColumn.columns)) {
           appendOnColumnChildren(operationCallback);
         } else {
-          const columnIndex = _.findIndex(tableColumns, getColumnSearchPredicate(selectedColumn));
-          const updatedColumns = operationCallback(tableColumns, columnIndex, generateNextAvailableColumn());
-          updateColumnsThenRows(operation, columnIndex, updatedColumns);
+          let columnIndex = -1;
+          for (const column of tableColumns as Array<ColumnInstance>) {
+            const foundIndex = column.columns?.findIndex(getColumnSearchPredicate(selectedColumn));
+            if (column.columns && foundIndex !== undefined && foundIndex !== -1) {
```

Comment:
isn't this the same as:
```ts
if (column.columns && foundIndex >= 0) {
```

?

Reply:
No, it is not. Typescript doesn't allow to make a `number` check with a possible `undefined` variable.

This would be the same:
```tsx
if (column.columns && foundIndex !== undefined && foundIndex >= 0)
```